### ISSUE TITLE
Patch Policies Ommitted

### ISSUE THREAD

**Issue by BIG-RAT** [BIG-RAT/Prune#1]
Currently patch policies are not scanned for groups or packages in use, but should be scanned shortly.

**Comment by BIG-RAT** [comment]
... and it's going to take longer than I'd hoped.  Looks like I might be battling a product issue.  Perform a GET on a patch policy and request XML I get the following for scoped computer groups:
```
<computer_groups>
	<computer_group>
		<id>175</id>
		<name>Beta Testers</name>
	</computer_group>
	<computer_group>
		<id>171</id>
		<name>Site: alpha 1</name>
	</computer_group>
</computer_groups>
```
same GET but requesting JSON yields:
```
"computer_groups":{"computer_group":{"id":171,
"name":"Site: alpha 1"}},
```
Which is missing a group.

**Comment by BIG-RAT** [comment]
Current app (v1.0.5) scans patch policies for packages, still need to scan for groups.

**Comment by BIG-RAT** [comment]
resolved with v1.1.0
